apps/server/src/modules/tldraw/utils/callback.ts

```diff
@@ -0,0 +1,75 @@
+const http = require('http')
+const CALLBACK_URL = process.env.CALLBACK_URL ? new URL(process.env.CALLBACK_URL) : null
+const CALLBACK_TIMEOUT = process.env.CALLBACK_TIMEOUT || 5000
+const CALLBACK_OBJECTS = process.env.CALLBACK_OBJECTS ? JSON.parse(process.env.CALLBACK_OBJECTS) : {}
+
+exports.isCallbackSet = !!CALLBACK_URL
+
+/**
+ * @param {Uint8Array} update
+ * @param {any} origin
+ * @param {WSSharedDoc} doc
+ */
+exports.callbackHandler = (update, origin, doc) => {
+	const room = doc.name
+	const dataToSend = {
+		room,
+		data: {}
+	}
+	const sharedObjectList = Object.keys(CALLBACK_OBJECTS)
+	sharedObjectList.forEach(sharedObjectName => {
+		const sharedObjectType = CALLBACK_OBJECTS[sharedObjectName]
+		dataToSend.data[sharedObjectName] = {
+			type: sharedObjectType,
+			content: getContent(sharedObjectName, sharedObjectType, doc).toJSON()
+		}
+	})
+	callbackRequest(CALLBACK_URL, CALLBACK_TIMEOUT, dataToSend)
+}
+
+/**
+ * @param {URL} url
+ * @param {number} timeout
+ * @param {Object} data
+ */
+const callbackRequest = (url, timeout, data) => {
```

Comment:
Please use the _axios_ client (as it's used within other client-side code across this project) for this HTTP callback request.